lang/cs/Org.Apache.REEF.IO/FileSystem/Hadoop/HadoopFileSystem.cs

```diff
@@ -47,11 +47,20 @@ private HadoopFileSystem(HdfsCommandRunner commandRunner)
         }
 
         /// <summary>
-        /// The Prefix used for URIs on this FileSystem.
+        /// CReate Uri from a given file name
+        /// If the file doesn't contain the preFIx, add the prefix in front
+        /// otherwise, use it directly
         /// </summary>
-        public string UriPrefix
+        /// <param name="path"></param>
+        /// <returns></returns>
+        public Uri CreateUriForPath(string path)
```

Comment:
Drop the documentation here and document the interface instead. Then, add what's different here.